wicket/src/ui/main.rs

```diff
@@ -113,8 +147,17 @@ impl MainScreen {
                 }
             },
             e => {
+                let current_pane = self.sidebar.selected();
                 if self.sidebar.selected {
```

Comment:
Hm, the proximity of `self.sidebar.selected()` (which returns the currently-selected pane?) and `self.sidebar.selected` (which is a bool that means... actually I'm not sure - the sidebar is focused?) is a little confusing here. Longer names for one or both might be warranted.

Reply:
Totally agree. I noticed that last night and would have slapped my forhead, except it was already bruised from banging into a desk :D

Reply:
I'll change it in the update PR that I'm working on.